pkg/apis/autoscaling/v1alpha1/pa_types.go

```diff
@@ -99,8 +99,10 @@ const (
 	// PodAutoscalerConditionReady is set when the revision is starting to materialize
 	// runtime resources, and becomes true when those resources are ready.
 	PodAutoscalerConditionReady = apis.ConditionReady
-	// PodAutoscalerConditionActive is set when the PodAutoscaler's ScaleTargetRef is receiving traffic.
+	// PodAutoscalerConditionActive turns true when the PodAutoscaler's ScaleTargetRef can receive traffic.
```

Comment:
Why this change?
```suggestion
	// PodAutoscalerConditionActive becomes true when the PodAutoscaler's ScaleTargetRef can receive traffic.
```

Reply:
changed "turns" to "becomes" as you suggested.  The change is because the old one isn't accurate.    A PA is active without traffic too.